airbyte-integrations/connectors/destination-bigquery/build.gradle

```diff
@@ -47,12 +47,11 @@ dependencies {
     integrationTestJavaImplementation project(':airbyte-integrations:connectors:destination-bigquery')
 
     // TODO: declare typing-deduping as a CDK feature instead of importing from source.
-    implementation project(':airbyte-cdk:java:airbyte-cdk:typing-deduping')
-    integrationTestJavaImplementation testFixtures(project(':airbyte-cdk:java:airbyte-cdk:typing-deduping'))
+//    implementation project(':airbyte-cdk:java:airbyte-cdk:typing-deduping')
+//    integrationTestJavaImplementation testFixtures(project(':airbyte-cdk:java:airbyte-cdk:typing-deduping'))
```

Comment:
Just delete this, don't comment things out.

```suggestion
```

Reply:
temp commit to verify all tests are good before deleting 😅